app/resources/api/v2/feed_resource.rb

```diff
@@ -32,6 +33,8 @@ def self.records(options = {}, skip_save_request = false)
         after = filters.dig(:after, 0)
         after = Time.parse(after) unless after.blank?
         feed_id = filters.dig(:feed_id, 0).to_i
+        skip_save_request = filters.dig(:skip_save_request, 0) == 'true'
```

Comment:
This method (`def sef.records`) already receives a `skip_save_request` argument... so you need to conciliate the two here, no? Something like:

```ruby
skip_save_request = skip_save_request_arg || filters.dig(:skip_save_request, 0) == 'true'
```

Reply:
Thank you, done!